storey/flow.py

```diff
@@ -395,6 +395,22 @@ async def _do(self, event):
             await self._do_internal(event, fn_result)
 
 
+class DropColumns(Flow):
+    def __init__(self, columns, **kwargs):
+        super().__init__(**kwargs)
+        if not isinstance(columns, list):
+            columns = [columns]
+        self._columns = columns
```

Comment:
Don't you need to force `full_event` here? If this step is added without `full_event` the `_do` function will fail, won't it?

Reply:
Using `full_event` or not is up to the step implementation. This step does not use `full_event` at all.